### ISSUE TITLE
[ADMIN] Asset Structure

### ISSUE THREAD

**Issue by gojeffcho** [TEAMJEFF/unarmed#52]
Changing LVL1 asset structure is non-optimal due to the clusterfuck it creates and everything it breaks (e.g. scripts being lost on objects).  However, this is the format all scenes and the overall project will follow from now on, starting from whenever we refactor our current situation (needs massive @kierandowns involvement to do so):

```Assets
  - Levels
    -> LVL1
    -> Testbed
    -> ...    
  - Sound
    -> SFX
    -> BGM
  - Models
    -> Characters
      -> Eos
      -> ...
    -> Objects
      -> ...
  - Scripts
    -> Controllers
    -> Mechanics
  - UI Elements```

Anything else that needs to be added WRT assets that don't fall into these categories, please ask me on Slack so that we can keep control.  Make the naming of things, e.g. scripts, sensible and consistent as well.

As for Layers:
  
```Layers
  * 0: Default
  * 1: TransparentFX
  * 2: IgnoreRaycast
  * 3:
  * 4: Water
  * 5: UI
  * 6: 
  * 7: 
  * 8: Player
  * 9: Anchored
  * 10: Unanchored
  * 11: ```

I'm leaving the default shit in, but the cullingmask for the Repel Gun is now everything anyway ("raycast everything and let the script sort it out").  The only two types of objects that there should be, as concerns _layers_, is whether it is anchored or unanchored.  If you are doing anything else with layers, it is probably wrong.  Again, if you need to add layers or change the ordering, please check with me first so it doesn't colossally buttfuck everything.

**Comment by gojeffcho** [comment]
TODO: tags

**Comment by gojeffcho** [comment]
Merged #56
